src/types.js

```diff
@@ -153,6 +153,8 @@ export type Dependency = {
   latest: string,
   url: string,
   hint: ?string,
+  range: string,
+  latestPattern: string,
```

Comment:
yeah, I agree... I had a hard time coming up with a meaningful name. Any suggestions? It's basically the `name@version` that we will upgrade to, which gets passed as input to the `add` command. Maybe something like `upgradeToPattern` or just `upgradeTo`?